plugins/kubernetes/app/models/kubernetes/service.rb

```diff
@@ -0,0 +1,95 @@
+module Kubernetes
+  # Convenience wrapper around the 'Service' object we can get from the K8S API
+  class Service
+    attr_reader :deploy_group, :role, :name, :ip_address, :port_info
+
+    def initialize(deploy_group: nil, role: nil)
+      @deploy_group = deploy_group
+      @role = role
+      @port_info = []
+      @name = role.try(:service_name)
```

Comment:
Service is now coupled with Role just to get the service name (role doesn't seem to be used anywhere else). Would it be simpler just to pass the name to the constructor?